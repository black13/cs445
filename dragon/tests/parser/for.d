program for_test();
begin
    for i := 0 to 10 do
        write(i)
end.
