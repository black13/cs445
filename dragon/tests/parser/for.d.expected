PROGRAM `for_test` WITH ARGS:
AND VARIABLES:
AND SUBPROGRAMS:
DOES:
  FOR `i` STARTING AT:
    LIT `0`
  AND GOING TO:
    LIT `10`
  DO:
    APPLY `write` WITH:
      i
-- done dumping ast --
